pkg/docker/build.go

```diff
@@ -12,6 +15,7 @@ ARG BUNDLE_DIR
 RUN apt-get update && apt-get install -y curl ca-certificates
 
 ARG DOCKER_VERSION=19.03.8
+RUN curl -o docker.tgz https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_VERSION}.tgz && \
```

Comment:
This added line appears to duplicate the one below, right?

Reply:
On line 17 above, I think `ARG DOCKER_VERSION=19.03.8` should be updated to `ARG DOCKER_VERSION=%s` now that we intend to use the string as a template with *printf.

Reply:
thank you, that makes sense!